### PR TITLE
Update rosdep with Kinetic dependencies

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -893,7 +897,7 @@ python-mako:
   ubuntu: [python-mako]
 python-mapnik:
   fedora: [mapnik-python]
-  ubuntu: [python-mapnik2]
+  ubuntu: [python-mapnik]
```
Comment (jacquelinekay) [inline rosdep/python.yaml]: Does this need to be separated out into the different supported Ubuntu versions? Or is `python-mapnik` actually used on all versions, not just Xenial?

Comment (cberner) [inline rosdep/python.yaml]: `python-mapnik` should work for all versions. `python-mapnik2` only exists for Trusty and there it's just a transitional package that depends on `python-mapnik`

Comment (jacquelinekay) [inline rosdep/python.yaml]: Great.

**Comment by jacquelinekay** [comment]
+1

**Comment by tfoote** [comment]
lgtm python-mapnik in ubuntu: http://packages.ubuntu.com/trusty/python/python-mapnik

**Pull request merged**